src/tufts/lib.js

```diff
@@ -195,5 +199,20 @@ class ImportData {
       }
     }
   }
+
+  /**
+   * Overrides feature data from an inflection with feature data from the lemma
+   * only applies to engine-specific list of featureTypes
+   * @param {String} featureType the feature type name
+   * @param {Inflection} inflection the inflection object
+   * @param {Lemma[]} lemmas the lemma objects
+   */
+  overrideInflectionFeature (featureType, inflection, lemmas) {
```

Comment:
I'm thinking that maybe a function name is a little misleading. From its name, it seems that it always override the feature (at least I would think this way), but it does so only for certain feature types. So I'm thinking maybe it's better to put a condition check outside so the function will always override as the name suggests:
```
if (needCorrection(featureType) { overrideInflectionFeature() }
```
or maybe rename it to reflect that overriding is conditional with something like `overrideFeatureIfIncorrect` or some other way.

I think with this small change could would be easier to read. Not to be nitpicking, I hope :-)

Reply:
hmm. that's a good suggestion. I prefer to keep the check in the function, but will rename it to something more accurate.